src/NServiceBus.AcceptanceTests/Recoverability/Retries/When_performing_slr_and_counting.cs

```diff
@@ -50,16 +50,14 @@ public RetryEndpoint()
             class ErrorNotificationSpy : IWantToRunWhenBusStartsAndStops
             {
                 Context testContext;
-                BusNotifications notifications;
+                Notifications notifications;
```

Comment:
I made a note about cleaning up the tests since there are also many which need namespace cleanup. Will make a dedicated PR.